lfroms/COMP4109-Project
# Users can send a message in a room

### What is the problem being solved in this PR?



Users can send a message in a room, it can't be displayed to the webpage 
### Related issues or PRs



Resolves #12 

### How did you accomplish this and why did you choose this approach?


socket.io

### How to test changes



1. yarn start:dev
2. Go to localhost:3000 and open the terminal on your local machine 
3. Go to  localhost:3000/chat/:id
4. send a message
5. check to see the message printed to the terminal
### Checklist
